### DISCUSSION TITLE
Not able to run  uniswap code locally.

### DISCUSSION THREAD

**Discussion by saroojbkhari** [Uniswap/interface#3201] [Q&A]
Hi all,
After getting latest code and try to run it locally, it's giving me following error.
can you please, and yes this file is not found in locales directory.

./src/constants/locales.ts
Module not found: Can't resolve '../locales/en-US'

**Comment by dracarys55** [comment]
Face the same problem.
Did you solve it ?

**Comment by vito-gmi** [comment]
Nearly 3 years later, I face the same problem.

@saroojbkhari @dracarys55, did you solve this?
